core/build.gradle

```diff
@@ -17,10 +17,18 @@ apply plugin: 'com.android.library'
 apply from: rootProject.file("configuration/checkstyle.gradle")
 apply from: rootProject.file("configuration/publishing.gradle")
 
+android {
+    compileOptions {
+        coreLibraryDesugaringEnabled true
+    }
+}
+
 dependencies {
+    coreLibraryDesugaring dependency.android.desugar_jdk_libs
+
```

Comment:
We can move these into the `afterEvaluate` block inside the root project `build.gradle`, so that they get applied to all modules, and keep the module `build.gradle` slim and consistent.

Reply:
Good idea, updated!